backend/.env.example

```
@@ -0,0 +1,13 @@
+# 服务器配置
+SERVER_PORT=3001
+CORS_ALLOW_ORIGIN=http://localhost:8080 # 前端项目地址
+
+# 默认路径配置
+DEFAULT_OUTPUT_DIR=output-log
+DEFAULT_SCHEMA_LOG_DIR=schema-log
+DEFAULT_API_LOG_DIR=raw-api-log
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DEFAULT_API_LOG_DIR key should go before the DEFAULT_OUTPUT_DIR key

(UnorderedKey)